meta-xt-prod-cockpit-rcar-driver-domain/recipes-connectivity/xen-network/files/eth0.network

```diff
@@ -0,0 +1,2 @@
+[Match]
```

Comment:
Hi @iusyk,

I've checked other similar implementations.

In meta-xt-common the network file is specified in SRC_URI and FILES sections:

https://github.com/xen-troops/meta-xt-common/blob/09d7a4c5fd4ebff2f78129f6798af1f4d699b26d/meta-xt-driver-domain/recipes-connectivity/xen-network/xen-network.bb#L11

https://github.com/xen-troops/meta-xt-common/blob/09d7a4c5fd4ebff2f78129f6798af1f4d699b26d/meta-xt-driver-domain/recipes-connectivity/xen-network/xen-network.bb#L26

Are you sure that we don't need to do the same thins in this change?

Reply:
Hi @iusyk, You've explained the logic of the changes to this file. This topic is closed.